jsk_recognition_msgs/action/ClipTask.action

```diff
@@ -0,0 +1,7 @@
+sensor_msgs/Image image
+string[] queries
+---
+jsk_recognition_msgs/ClipResult result
```

Comment:
We must be very careful on 'common noun' vs 'proper noun'
In genral, the service / task will be provided as `common noun`, because pople do not care about actual imprementation.

```
  QueryAndProbability.msg
  QuestionAndAnswerText.msg
```
This is seems good. I personally prefer `QustionAndAnswer` and `QueryAndProbability` or `QustionAndAnserText` and `QueryAndProbabilityXxxxx`.
```
  VQAResult.msg
  ClipResult.msg
```
VQA is common noun, but `Clip` is proper noun. So it should be VQAResult and VQPResult? ObjectDetetion? OpenVocavularyObjectDetection? The name of `QueryAndProbability.msg` should be update if you use other than VAP.
```
  VQATask.action
  ClipTask.action
```
Same as `Result.msg`